packages/issuer/package.json

```diff
@@ -15,7 +15,9 @@
   },
   "devDependencies": {
     "@sphereon/openid4vci-client": "workspace:../client",
+    "@sphereon/openid4vci-issuer-callback": "workspace:../issuer-callback",
```

Comment:
I would expect the deps to be the other way around

Reply:
Yes, I've made it issuer centric. Fixed